validation/lst1-dl3-crab/make.py

```diff
@@ -0,0 +1,215 @@
+import logging
+import time
+from pathlib import Path
+from astropy.coordinates import SkyCoord
+
+import click
+import astropy.units as u
+
+from gammapy.data import DataStore
+from gammapy.datasets import Datasets, SpectrumDataset
+from gammapy.estimators import FluxPointsEstimator
+from gammapy.makers import (
+    SpectrumDatasetMaker,
+    WobbleRegionsFinder,
+    ReflectedRegionsBackgroundMaker,
+)
+from gammapy.maps import MapAxis, RegionGeom
+from gammapy.modeling import Fit
+from gammapy.modeling.models import (
+    SkyModel,
+    LogParabolaSpectralModel
+)
+from regions import PointSkyRegion
+import requests
+
+
+log = logging.getLogger(__name__)
+
+LST1_WORKING_DIR = Path(__file__).parent
+DL3_PATH = LST1_WORKING_DIR / "data"
+PATH_RESULTS = LST1_WORKING_DIR / "results"
+REFERENCE_RESULTS = LST1_WORKING_DIR / "reference"
+
+ZENODO_URL_LST1_CRAB = '11445184'
+
+
+@click.group()
+@click.option("--log-level", default="INFO", type=click.Choice(["DEBUG", "INFO", "WARNING"]))
+def cli(log_level):
+    logging.basicConfig(level=log_level)
+    get_data_from_zenodo()
+
+
+def dl3_to_dl4_reduction(dl3_path):
+    """DL3 to DL4 reduction. Returns a Datasets object."""
+    datastore = DataStore.from_dir(dl3_path)
+    observations = datastore.get_observations(required_irf="point-like")
+    target_position = SkyCoord.from_name("Crab Nebula", frame='icrs')
```

Comment:
Maybe provide the coordinates to avoid the query everytime

Reply:
done